src/SFA.DAS.Approvals/InnerApi/Requests/PostValidateBulkUploadRequest.cs

```diff
@@ -0,0 +1,18 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.Approvals.InnerApi.Requests
+{
+    public class PostValidateBulkUploadRequest : IPostApiRequest
+    {
+        public string PostUrl => $"api/{ProviderId}/bulkupload/validate";
+
+        public PostValidateBulkUploadRequest(long providerId, BulkUploadValidateApiRequest data)
+        {
+            ProviderId = providerId;
+            Data = data;
+        }
+
+        public long ProviderId { get; }
```

Comment:
this property is unused, also could we consistently place all properties above the constructor?